miniml/compiler/test/arith.ml

```diff
@@ -0,0 +1,21 @@
+open Lib
```

Comment:
Could we use the new open functionality to avoid opening `Lib` in all files?

Reply:
Yes, once it is available in `master`.

Reply:
It is now.

Reply:
Done!